localstack-core/localstack/testing/pytest/fixtures.py

```diff
@@ -1663,7 +1663,10 @@ def _create_parameter(**kwargs):
     yield _create_parameter
 
     for param in params:
-        aws_client.ssm.delete_parameter(Name=param)
+        try:
+            aws_client.ssm.delete_parameter(Name=param)
+        except Exception as e:
+            LOG.debug("error cleaning up parameter %s: %s", param, e)
```

Comment:
When I used the existing `create_parameter` method, I got the following error. Because the test calling create_parameter three times with the Overwrite=True parameter.

```
ERROR tests/aws/services/ssm/test_ssm.py::TestSSM::test_get_parameter_history - botocore.errorfactory.ParameterNotFound: An error occurred (ParameterNotFound) when calling the DeleteParameter operation: Parameter param-36e2b5a6 not found.
```

Since the parameter only needs to be deleted once, I added exception handling, referencing the existing implementation😀

https://github.com/localstack/localstack/blob/master/localstack-core/localstack/testing/pytest/fixtures.py#L163-L164